infra/azure/nightly.yml

```diff
@@ -10,7 +10,7 @@ schedules:
 trigger: none
 
 pool:
-  vmImage: 'ubuntu-20.04'
+  vmImage: 'ubuntu-22.04'
```

Comment:
Is this working with the multi item tests?

Reply:
This change should not make into the PR. Fixed.